presto-docs/src/main/sphinx/release/release-328.rst

```diff
@@ -0,0 +1,63 @@
+========================
+Release 328 (6 Jan 2020)
+========================
+
+General Changes
+---------------
+
+* Fix correctness issue for certain correlated JOIN queries when the subquery on the
+  right produces no rows. (:issue:`1969`)
+* Fix incorrect handling of multi-byte characters when pattern is empty. (:issue:`2313`)
+* Fix failure when join criteria contains columns of different types. (:issue:`2320`)
+* Fix failure for complex outer join queries when dynamic filtering is enabled. (:issue:`2363`)
+* Improve support for correlated queries. (:issue:`1969`)
+* Support narrowing type coercions for insert queries. (:issue:`2061`)
+* Add ``regexp_count`` and ``regexp_position`` SQL functions. (:issue:`2136`)
+
+Security Changes
+----------------
+
+* Fix a security issue allowing users to contain unauthorized access to Presto cluster
```

Comment:
This doesn't make sense (and the entire thing isn't grammatically correct):

> allowing users to contain unauthorized access

Is this supposed to be "gain"? We should be more specific about the issue.

Reply:
Yes, I believe that's accurate. @findepi?

Reply:
Yes, "gain". 

> We should be more specific about the issue.

We could expand description of the existing issue and rely on the issue link for providing more information.